spec/features/consumer/account/payments_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+feature "Payments requiring action", js: true do
+  include AuthenticationHelper
+
+  describe "as a logged in user" do
+    let(:user) { create(:user) }
+    let(:order) { create(:order, user: user) }
+
+    before do
+      login_as user
+    end
+
+    context "there is a payment requiring authorization" do
+      let!(:payment) do
+        create(:payment, order: order, cvv_response_message: "https://stripe.com/redirect")
+      end
+
+      it "shows a table of payments requiring authorization" do
+        visit "/account"
+
+        find("a", :text => %r{#{I18n.t('spree.users.show.tabs.transactions')}}i).click
```

Comment:
That's a great question. I tried switching to click_link and it doesn't find it. My guess is it's something to do with the fact that they're tabs with anchor links instead of links to full URL's, but that's just a guess.